source-code/core/app/inlang.config.json

```diff
@@ -0,0 +1 @@
+{"sourceLanguageTag":"en","languageTags":["en"],"modules":["./dist/index.js"],"settings":{"plugins":{"inlang.plugin.i18next":{"options":{"pathPattern":"./examples/example01/{languageTag}.json","variableReferencePattern":["{","}"]}}},"lintRules":{"inlang.missingMessage":{"level":"error"}}}}
```

Comment:
Why do we have this file here?

Reply:
Always wondered that. I think @NilsJacobsen used it for testing. Let's remove this file